affirm/src/main/java/com/affirm/android/AffirmPlugins.java

```diff
@@ -15,7 +15,7 @@
 import okhttp3.OkHttpClient;
 import okhttp3.Request;
 
-class AffirmPlugins {
+public class AffirmPlugins {
```

Comment:
isn't `public` the default in Java?

Reply:
No, default in Java can only be accessed in the same package.  https://www.javatpoint.com/access-modifiers